renderdoc/driver/vulkan/vk_resources.h

```diff
@@ -1090,6 +1090,7 @@ struct MemMapState
   bool mapCoherent;
   byte *mappedPtr;
   byte *refData;
+  Threading::SpinLock mrLock;
```

Comment:
We shouldn't use a spinlock here. In general they should be avoided and I don't think there's any reason we need one here.